<?xml version="1.0" encoding="utf-8"?>
<xliff xmlns="urn:oasis:names:tc:xliff:document:1.2" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" version="1.2" xsi:schemaLocation="urn:oasis:names:tc:xliff:document:1.2 xliff-core-1.2-transitional.xsd">
  <file datatype="xml" original="XpsPackagingException.xml" source-language="en-US" target-language="de-DE">
    <header>
      <tool tool-id="mdxliff" tool-name="mdxliff" tool-version="1.0-15c36f0" tool-company="Microsoft" />
      <xliffext:skl_file_name xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">02cd5861-7ce2-4a82-b358-31f8435a0ac5b0d57cc01958129070a222918b0a5a52fc7844df.skl</xliffext:skl_file_name>
      <xliffext:version xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">1.2</xliffext:version>
      <xliffext:ms.openlocfilehash xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">b0d57cc01958129070a222918b0a5a52fc7844df</xliffext:ms.openlocfilehash>
      <xliffext:ms.sourcegitcommit xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">d31dc2ede16f6f7bc64e90d9f897ff54c4e3869b</xliffext:ms.sourcegitcommit>
      <xliffext:ms.lasthandoff xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">04/03/2018</xliffext:ms.lasthandoff>
      <xliffext:moniker_ids xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">netframework-4.5.1,netframework-4.5.2,netframework-4.5,netframework-4.6.1,netframework-4.6.2,netframework-4.6,netframework-4.7.1,netframework-4.7</xliffext:moniker_ids>
    </header>
    <body>
      <group id="content" extype="content">
        <trans-unit id="101" translate="yes" xml:space="preserve" uid="T:System.Windows.Xps.XpsPackagingException">
          <source>The exception that is thrown when reading, writing to, registering, or accessing in some other way an <ph id="ph1">&lt;see cref="T:System.Windows.Xps.Packaging.XpsDocument" /&gt;</ph>.</source>
          <target state="translated">Die Ausnahme, die ausgelöst wird, wenn in einem <ph id="ph1">&lt;see cref="T:System.Windows.Xps.Packaging.XpsDocument" /&gt;</ph> gelesen oder geschrieben wird, wenn es registriert wird oder wenn auf andere Weise darauf zugegriffen wird.</target>       </trans-unit>
        <trans-unit id="102" translate="yes" xml:space="preserve" uid="T:System.Windows.Xps.XpsPackagingException">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:System.Windows.Xps.XpsPackagingException" /&gt;</ph> class.</source>
          <target state="translated">Initialisiert eine neue Instanz der <ph id="ph1">&lt;see cref="T:System.Windows.Xps.XpsPackagingException" /&gt;</ph>-Klasse.</target>       </trans-unit>
        <trans-unit id="103" translate="yes" xml:space="preserve" uid="M:System.Windows.Xps.XpsPackagingException.#ctor">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:System.Windows.Xps.XpsPackagingException" /&gt;</ph> class.</source>
          <target state="translated">Initialisiert eine neue Instanz der <ph id="ph1">&lt;see cref="T:System.Windows.Xps.XpsPackagingException" /&gt;</ph>-Klasse.</target>       </trans-unit>
        <trans-unit id="104" translate="yes" xml:space="preserve" uid="M:System.Windows.Xps.XpsPackagingException.#ctor(System.String)">
          <source>A <ph id="ph1">&lt;see cref="T:System.String" /&gt;</ph> that describes the error condition.</source>
          <target state="translated">Ein <ph id="ph1">&lt;see cref="T:System.String" /&gt;</ph>, der den Fehlerzustand beschreibt.</target>       </trans-unit>
        <trans-unit id="105" translate="yes" xml:space="preserve" uid="M:System.Windows.Xps.XpsPackagingException.#ctor(System.String)">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:System.Windows.Xps.XpsPackagingException" /&gt;</ph> class that provides a specific error condition.</source>
          <target state="translated">Initialisiert eine neue Instanz der <ph id="ph1">&lt;see cref="T:System.Windows.Xps.XpsPackagingException" /&gt;</ph>-Klasse, die einen bestimmten Fehlerzustand bereitstellt.</target>       </trans-unit>
        <trans-unit id="106" translate="yes" xml:space="preserve" uid="M:System.Windows.Xps.XpsPackagingException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)">
          <source>The data that is required to serialize or deserialize an object.</source>
          <target state="translated">Die zum Serialisieren bzw. Deserialisieren eines Objekts benötigten Daten.</target>       </trans-unit>
        <trans-unit id="107" translate="yes" xml:space="preserve" uid="M:System.Windows.Xps.XpsPackagingException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)">
          <source>The context, which includes source and destination, of the serialized stream.</source>
          <target state="translated">Der Kontext des serialisierten Streams, einschließlich der Quelle und des Ziels.</target>       </trans-unit>
        <trans-unit id="108" translate="yes" xml:space="preserve" uid="M:System.Windows.Xps.XpsPackagingException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:System.Windows.Xps.XpsPackagingException" /&gt;</ph> class that provides specific <ph id="ph2">&lt;see cref="T:System.Runtime.Serialization.SerializationInfo" /&gt;</ph> and <ph id="ph3">&lt;see cref="T:System.Runtime.Serialization.StreamingContext" /&gt;</ph>.</source>
          <target state="translated">Initialisiert eine neue Instanz der <ph id="ph1">&lt;see cref="T:System.Windows.Xps.XpsPackagingException" /&gt;</ph>-Klasse, die eine bestimmte <ph id="ph2">&lt;see cref="T:System.Runtime.Serialization.SerializationInfo" /&gt;</ph> und einen <ph id="ph3">&lt;see cref="T:System.Runtime.Serialization.StreamingContext" /&gt;</ph> bereitstellt.</target>       </trans-unit>
        <trans-unit id="109" translate="yes" xml:space="preserve" uid="M:System.Windows.Xps.XpsPackagingException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)">
          <source>This constructor is protected.</source>
          <target state="translated">Dieser Konstruktor ist geschützt.</target>       </trans-unit>
        <trans-unit id="110" translate="yes" xml:space="preserve" uid="M:System.Windows.Xps.XpsPackagingException.#ctor(System.String,System.Exception)">
          <source>A <ph id="ph1">&lt;see cref="T:System.String" /&gt;</ph> that describes the error condition.</source>
          <target state="translated">Ein <ph id="ph1">&lt;see cref="T:System.String" /&gt;</ph>, der den Fehlerzustand beschreibt.</target>       </trans-unit>
        <trans-unit id="111" translate="yes" xml:space="preserve" uid="M:System.Windows.Xps.XpsPackagingException.#ctor(System.String,System.Exception)">
          <source>The underlying error that caused the <ph id="ph1">&lt;see cref="T:System.Windows.Xps.XpsPackagingException" /&gt;</ph>.</source>
          <target state="translated">Der zugrunde liegende Fehler, durch den <ph id="ph1">&lt;see cref="T:System.Windows.Xps.XpsPackagingException" /&gt;</ph> ausgelöst wurde.</target>       </trans-unit>
        <trans-unit id="112" translate="yes" xml:space="preserve" uid="M:System.Windows.Xps.XpsPackagingException.#ctor(System.String,System.Exception)">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:System.Windows.Xps.XpsPackagingException" /&gt;</ph> class that provides a specific error condition and includes the cause of the exception.</source>
          <target state="translated">Initialisiert eine neue Instanz der <ph id="ph1">&lt;see cref="T:System.Windows.Xps.XpsPackagingException" /&gt;</ph>-Klasse, die einen bestimmten Fehlerzustand bereitstellt und die Ursache für die Ausnahme umfasst.</target>       </trans-unit>
        <trans-unit id="113" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Windows.Xps.XpsPackagingException.#ctor(System.String,System.Exception)">
          <source>If the <ph id="ph1">`innerException`</ph> parameter is not <ph id="ph2">`null`</ph>, the current exception is thrown in a <ph id="ph3">`catch`</ph> block that handles the inner exception.</source>
          <target state="translated">Wenn die <ph id="ph1">`innerException`</ph> -Parameter ist kein <ph id="ph2">`null`</ph>, die aktuelle Ausnahme wird ausgelöst, eine <ph id="ph3">`catch`</ph> Block, der die innere Ausnahme behandelt.</target>       </trans-unit>
      </group>
    </body>
  </file>
</xliff>